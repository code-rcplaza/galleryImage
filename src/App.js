import React, { Fragment, useState } from 'react';
import ImageGrid from './components/ImageGrid';

import './App.css';
import { NavBar } from './components/NavBar';
import { TitlePage } from './components/TitlePage'

const App = ({ defaultCategory = ['photos'] }) => {

  const [categories /*, setCategories*/] = useState(defaultCategory)


  return (
    <Fragment>
      <NavBar />

      <TitlePage />

      {categories.map((category) => (
        <ImageGrid key={category}
          category={category} />
      ))}
    </Fragment>
  );
}

export default App