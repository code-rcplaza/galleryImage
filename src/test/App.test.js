import { shallow } from 'enzyme';
import React from 'react';
import App from '../App';

describe('Tests on <App />', () => {
    
    test('Should be correctly render ', () => {
        const wrapper = shallow(<App />)
    
        expect( wrapper ).toMatchSnapshot()
    })

})