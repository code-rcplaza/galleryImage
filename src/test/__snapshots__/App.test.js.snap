// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on <App /> Should be correctly render  1`] = `
<Fragment>
  <NavBar />
  <TitlePage />
  <ImageGrid
    category="photos"
    key="photos"
  />
</Fragment>
`;
